ImportBlocks/ImportBlocks.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Neo" Version="2.9.4" />
+    <PackageReference Include="Neo" Version="2.10.1" />
```

Comment:
Can't be pushed anyway until this reference is available.

Reply:
I can update this to a NuGet build, once https://github.com/neo-project/neo/pull/633 is pushed.

Reply:
That is it, Jeff.